pkg/api/config.go

```diff
@@ -229,8 +230,60 @@ func validateReleaseBuildConfiguration(input *ReleaseBuildConfiguration) error {
 		validationErrors = append(validationErrors, errors.New("`base_rpm_images` defined but no `rpm_build_commands` found"))
 	}
 
-	if input.Resources == nil {
-		validationErrors = append(validationErrors, errors.New("`resources` cannot be empty, at least the blanket `*` has to be specified"))
+	validationErrors = append(validationErrors, validateResources("resources", input.Resources))
+	return kerrors.NewAggregate(validationErrors)
+}
+
+func validateResources(fieldRoot string, resources ResourceConfiguration) error {
+	var validationErrors []error
+	if resources == nil {
+		validationErrors = append(validationErrors, fmt.Errorf("`%s` cannot be empty", fieldRoot))
+	} else {
+		if _, exists := resources["*"]; !exists {
+			validationErrors = append(validationErrors, fmt.Errorf("`%s` must specify a blanket policy for `*`", fieldRoot))
+		}
+		for key := range resources {
+			validationErrors = append(validationErrors, validateResourceRequirements(fmt.Sprintf("%s.%s", fieldRoot, key), resources[key]))
+		}
+	}
+
+	return kerrors.NewAggregate(validationErrors)
+}
+
+func validateResourceRequirements(fieldRoot string, requirements ResourceRequirements) error {
+	var validationErrors []error
+
+	validationErrors = append(validationErrors, validateResourceList(fmt.Sprintf("%s.limits", fieldRoot), requirements.Limits))
+	validationErrors = append(validationErrors, validateResourceList(fmt.Sprintf("%s.requests", fieldRoot), requirements.Requests))
+
+	return kerrors.NewAggregate(validationErrors)
+}
+
+func validateResourceList(fieldRoot string, list ResourceList) error {
+	var validationErrors []error
+
+	var numInvalid int
+	for key := range list {
+		switch key {
+		case "cpu", "memory":
+			if quantity, err := resource.ParseQuantity(list[key]); err != nil {
```

Comment:
Do we need to validate a minimum value? For example, in this case, if the user will put `5` instead of `5Gi` we will assume that the value is only 5 bytes. 
Also if the user will input `5Gb` instead of `5Gi` the value will be `0` which is validated below. Do we need to validate also the `Ki | Mi | Gi | Ti | Pi | Ei ` formatting?

Reply:
> Do we need to validate a minimum value? 

I don't think it's super useful. If the user wants to shoot themselves in the foot (in a way that won't crash the ci-operator) so be it ...

> Do we need to validate also the Ki | Mi | Gi | Ti | Pi | Ei formatting?

The parser will do it

Reply:
> Also if the user will input 5Gb instead of 5Gi the value will be 0 which is validated below

No, that's an invalid suffix

```yaml
resources:
  '*':
    limits:
      cpu: '6'
      memory: 3Gb
    requests:
      cpu: '3'
      memory: 1Gi
```

```
error loading configuration files: invalid configuration from test.yaml: resources.*.limits.memory: invalid quantity: quantities must match the regular expression '^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
```